org.eclipse.m2e.repository/pom.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+	Copyright (c) 2008, 2022 Sonatype, Inc. and others
+	All rights reserved. This program and the accompanying materials
+	are made available under the terms of the Eclipse Public License 2.0
+	which accompanies this distribution, and is available at
+	https://www.eclipse.org/legal/epl-2.0/
+
+	SPDX-License-Identifier: EPL-2.0
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.eclipse.m2e</groupId>
+		<artifactId>m2e-core</artifactId>
+		<version>2.0.0-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>org.eclipse.m2e.repository</artifactId>
+	<packaging>eclipse-repository</packaging>
+
+	<name>Maven Integration for Eclipse Repository</name>
+
+	<profiles>
+		<!-- Eclipse SimRel requires all artifacts to be jar- or pgp-signed. So we sign the (non eclipse) artifacts
+			and update the p2 metadata accordingly -->
+		<profile>
+			<id>eclipse-sign</id>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.eclipse.tycho</groupId>
+						<artifactId>tycho-gpg-plugin</artifactId>
+						<version>${tycho-version}</version>
+						<executions>
+							<execution>
+								<id>pgpsigner</id>
+								<goals>
+									<goal>sign-p2-artifacts</goal>
+								</goals>
+								<configuration>
+									<keyname>${gpg.keyname}</keyname>
```

Comment:
If the `tycho-gpg-plugin` would have a skip-property this pom could be replaced by a profile in the root pom that is activated for a category.xml file and skips this execution if the eclipse-sign profile is not activated (by setting a property like `pgp.signing.skip` that is true by default and false in that profile.

Reply:
To be honest, I don't think it would be really better...

Reply:
> To be honest, I don't think it would be really better...

Yes, after thinking about it again, I agree. Since it is no problem to for example adjust the parent's version in this pom, it is not worth to try hard to get rid of it.